Description: Blanks for tutorial related dll after repacking
 Replace prebuilt dlls with blanks until they can be buildable.
Author: Obey Arthur Liu <arthur@milliways.fr>
Last-Update: 2009-12-29
--- /dev/null
+++ b/binsrc/tutorial/bin/tabcontrol.dll
@@ -0,0 +1 @@
+# Debian maintainer: intentionally blank
--- /dev/null
+++ b/binsrc/tutorial/bin/tabcontrol2.dll
@@ -0,0 +1 @@
+# Debian maintainer: intentionally blank
--- /dev/null
+++ b/binsrc/tutorial/hosting/ho_s_10/Point_ho_s_10.dll
@@ -0,0 +1 @@
+# Debian maintainer: intentionally blank
--- /dev/null
+++ b/binsrc/tutorial/hosting/ho_s_11/restricted.dll
@@ -0,0 +1 @@
+# Debian maintainer: intentionally blank
--- /dev/null
+++ b/binsrc/tutorial/hosting/ho_s_11/unrestricted.dll
@@ -0,0 +1 @@
+# Debian maintainer: intentionally blank
--- /dev/null
+++ b/binsrc/tutorial/hosting/ho_s_12/bin/WebService15.dll
@@ -0,0 +1 @@
+# Debian maintainer: intentionally blank
--- /dev/null
+++ b/binsrc/tutorial/hosting/ho_s_14/ho_s_14.dll
@@ -0,0 +1 @@
+# Debian maintainer: intentionally blank
--- /dev/null
+++ b/binsrc/tutorial/hosting/ho_s_15/COM/VirtCOMServer/Debug/VirtCOMServer.dll
@@ -0,0 +1 @@
+# Debian maintainer: intentionally blank
--- /dev/null
+++ b/binsrc/tutorial/hosting/ho_s_15/COM/VirtCOMServer/VirtCOMServer.dll
@@ -0,0 +1 @@
+# Debian maintainer: intentionally blank
--- /dev/null
+++ b/binsrc/tutorial/hosting/ho_s_2/Point.dll
@@ -0,0 +1 @@
+# Debian maintainer: intentionally blank
--- /dev/null
+++ b/binsrc/tutorial/hosting/ho_s_2/tax.dll
@@ -0,0 +1 @@
+# Debian maintainer: intentionally blank
--- /dev/null
+++ b/binsrc/tutorial/hosting/ho_s_3/redcoalsms.dll
@@ -0,0 +1 @@
+# Debian maintainer: intentionally blank
--- /dev/null
+++ b/binsrc/tutorial/hosting/ho_s_4/redcoalsms.dll
@@ -0,0 +1 @@
+# Debian maintainer: intentionally blank
--- /dev/null
+++ b/binsrc/tutorial/hosting/ho_s_5/redcoalsms_dom.dll
@@ -0,0 +1 @@
+# Debian maintainer: intentionally blank
--- /dev/null
+++ b/binsrc/tutorial/services/so_s_32/so_s_32.dll
@@ -0,0 +1 @@
+# Debian maintainer: intentionally blank
